### DISCUSSION TITLE
Searchable collections in data model

### DISCUSSION THREAD

**Discussion by darioguarascio** [directus/directus#8870] [Feature Requests]
With the new collection visualization in data model, the overall height taken is much more than before, it requires scrolling to find out where your collection is, not to mention they are not sorted alphabetically anymore.
A quicker access would be taking back usabiliy from non-folder model, and it could be a filter input field, a sort button or anything like that

**Comment by benhaynes** [comment]
An interesting point. I wonder if we should sort them alphabetically if they have a `NULL` sort value. 🤔  @rijkvanzanten 

In any event, you can sort them alphabetically if you want... but have the option to create a more meaningful organization, which should make it _easier_ to find what you want.

**Reply by rijkvanzanten** [reply]
> I wonder if we should sort them alphabetically if they have a NULL sort value.

It does 👍🏻

**Comment by darioguarascio** [comment]
@benhaynes @rijkvanzanten perhaps the overall best would be a selectable layout. Like in collection or file view, there are list  table, cards etc, in this case it could be a folder-like visualization (very helpful when there are many collections - the case I'm dealing with now)

**Reply by benhaynes** [reply]
Unfortunately, this is not a standard Collection page, and this new UI is not a Layout... so that would be too difficult to do. Also, the main configuration of groups and nesting would only work with this UI. I'm curious how much different this really is though, as Data Model rows used to be 50px each, and now they're not much more than that. We could have an option that the height shrinks a bit when more than X collections are there... but this page isn't built for switching layouts.

Another option would be to add a search pane to the top to quickly filter them down... but that gets hard with nesting.

**Comment by darioguarascio** [comment]
@benhaynes and what about a flat table visualization that adds a `parent` column ?

**Reply by benhaynes** [reply]
I guess I just don't know the benefit of this, we can solve the vertical height by compressing the current UI. For the order, I would assume that the nested order you choose would be just as convenient as alphabetical, right? What I'm saying is there needs to be a clear problem to solve (that applies to 80% of our users... our 80/20 rule) before we start adding a lot of new code to build/maintain.

How many collections do you have? Are you having a hard time finding them (order/nesting), or just scrolling to them (height)? Any insight into _what we're trying to solve_ would be helpful

Also, you can respond to the previous thread to keep this organized. 😉

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
